### DISCUSSION TITLE
Is there any option for deadbeef to play in minimized?

### DISCUSSION THREAD

**Discussion by asgharam** [DeaDBeeF-Player/deadbeef#2591] [Q&A]
Hey. I wondered if there is an option for when you want to play a song with deadbeef, the program doesn't appear and starts playing in the background? if not, is there any workaround?

**Comment by Oleksiy-Yakovenko** [comment]
Unfortunately, there doesn't seem to be a built-in way or workaround for this.

**Reply by asgharam** [reply]
uh well, it would be cool if that feature gets implemented. Personally when I'm working and I'm playing a music, I don't want the music player gets in my way.
